src/main.cpp

```diff
@@ -61,11 +67,13 @@ void setup() {
 	can_bus.Initialize(ICAN::BaudRate::kBaud1M);
 
 	//Initialize our timer(s)
-	read_timer.AddTimer(100, ReadAcceleratorPress);
+	read_timer.AddTimer(100, getAcceleratorPress);
 	read_timer.AddTimer(100, printReceiveSignals);
 }
 
 void loop() {
-	delay(0);
+	delay(1000);
```

Comment:
why delay here?

Reply:
My bad, that was a leftover from testing